exchanges/zb/zb_test.go

```diff
@@ -390,7 +390,7 @@ func TestModifyOrder(t *testing.T) {
 	if mockTests {
 		t.Skip("skipping authenticated function for mock testing")
 	}
-	if z.ValidateAPICredentials() && !canManipulateRealOrders {
+	if z.AreCredentialsValid(context.Background()) && !canManipulateRealOrders {
```

Comment:
In zb_test, bitfinex_test and huobi_test, update these to use 
`ValidateAPICredentials(o.GetDefaultCredentials()) == nil`